snuba/subscriptions/data.py

```diff
@@ -9,7 +9,7 @@
 from snuba.request.request_settings import SubscriptionRequestSettings
 from snuba.request.schema import RequestSchema
 from snuba.utils.metrics.timer import Timer
-from snuba.views import validate_request_content
+from snuba.web.views import validate_request_content
```

Comment:
This seems overwhelmingly wrong. (I mean the place validate_request_query is, and thus this dependency - not the change in this PR).

Reply:
I agree, I think we are basically doomed to clean up this module for eternity like Sisyphus and his rock.